### ISSUE TITLE
hakchi2 extra options ?

### ISSUE THREAD

**Issue by ghost** [ClusterM/hakchi2#449]
Hi 

For nes mini / snes mini

Hakchi2
Main window delete all games that are selected/ticked only . currently you have to delete each game.
Main window download box art for selected/ticked games only. currently it downloads for all games. 
Create/name Folders in main window with games inside

Console
Nes/Snes home screen option do not scroll list of folders/consoles. keep folders centered home screen . yes scroll inside folders games

Just a thought not sure if its been asked a already :)

**Comment by zeusz4u** [comment]
I also asked the same quite a while ago in issue #235 .

@ClusterM confirmed it's planned to add these enhancements in the future. Probably he's quite busy now with adding support for the SNES.

I thought of one more enchancement that would also be cool to have: download box art for newly added games only (or only those that don't have a box art yet), don't search and override boxart for the games that already have box art in the game folder. It would be useful when you're adding batches of new games to hakchi and none of these have boxarts yet.

**Comment by ghost** [comment]
yes he will be very busy atm i understand . i agree with you on your options they would be very useful also .

**Comment by AmesJainchill** [comment]
Not certain if I should start a separate Feature Request for this or not and after a bit of searching I have not seen this request already, but since this topic is more general, I will start here.

I personally would love to see a feature implemented on NES and SNES Classic consoles via hakchi2 which would allow for emulation speed increase or decrease on-demand ideally, perhaps by fixed percentages via button press as is commonly available on PC nes/snes/etc. emulators.  For example, being able to advance emulation speed to say 200-400% when trying to gain experience in a lengthy game would be a great time saver especially for those of us who aren't 10 years old (as I was when I got my NES back in the 80's) and don't have hours to spend on games any longer.  : )

Thanks @ClusterM for all your hard work on this continuing project!
